src/encoded/static/components/award.js

```diff
@@ -409,24 +737,481 @@ class StatusChart extends React.Component {
+                this.chart = chartInstance;
+            });
+    }
+
+    render() {
+        const { statuses, ident } = this.props;
+
+        // Calculate a (hopefully) unique ID to put on the DOM elements.
+        const id = `${statusChartId}-${ident}-controls`;
+
+        return (
+            <div className="award-charts__chart">
+                <div className="award-charts__title">
+                    Controls
+                </div>
+                {statuses.length ?
+                    <div className="award-charts__visual">
+                        <div id={id} className="award-charts__canvas">
+                            <canvas id={`${id}-chart`} />
+                        </div>
+                        <div id={`${id}-legend`} className="award-charts__legend" />
+                    </div>
+                :
+                    <div className="chart-no-data" style={{ height: this.wrapperHeight }}>No data to display</div>
+                }
+            </div>
+        );
+    }
+}
+
+ControlsChart.propTypes = {
+    award: PropTypes.object.isRequired, // Award being displayed
+    statuses: PropTypes.array, // Array of status facet data
+    linkUri: PropTypes.string.isRequired, // URI to use for matrix links
+    ident: PropTypes.string.isRequired, // Unique identifier to `id` the charts
+    objectQuery: PropTypes.string,
+};
+
+ControlsChart.defaultProps = {
+    statuses: [],
+    objectQuery: '',
+};
+
+ControlsChart.contextTypes = {
+    navigate: PropTypes.func,
+};
+
+class StatusExperimentChart extends React.Component {
+    constructor() {
+        super();
+        this.createChart = this.createChart.bind(this);
+        this.updateChart = this.updateChart.bind(this);
+    }
+
+    componentDidMount() {
+        if (this.props.statuses.length) {
+            this.createChart(`${statusChartId}-${this.props.ident}`, this.props.statuses);
+        }
+    }
+
+    componentDidUpdate() {
+        if (this.props.statuses.length) {
+            if (this.chart) {
+                this.updateChart(this.chart, this.props.statuses);
+            } else {
+                this.createChart(`${statusChartId}-${this.props.ident}`, this.props.statuses);
+            }
+        } else if (this.chart) {
+            this.chart.destroy();
+            this.chart = null;
+        }
+    }
+
+    updateChart(chart) {
+        const { experiments, unreplicated, isogenic, anisogenic, linkUri, award, objectQuery } = this.props;
+        const data = StatusData(experiments, unreplicated, isogenic, anisogenic); // Array of datasets and labels
+        const replicatelabels = ['unreplicated', 'isogenic', 'anisogenic'];
+        const colors = replicatelabels.map((label, i) => statusColorList[i % statusColorList.length]);
+        const unreplicatedDataset = data[2];
+        const isogenicDataset = data[3];
+        const anisogenicDataset = data[4];
+        if (unreplicatedDataset.some(x => x > 0)) {
```

Comment:
this doesn't seem quite right either.   LIke instead of 3 variables for the different types of Datasets, use an object with a property "datasetType"?    It's quite possible I am missing something though.